portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+    <body>
+        <header>
+        <div id ="header">
+            <h1 style="text-align:center"> My Projects </h1>
+        <div id="padding">
+            <a href="index.html"> Home </a>
+            </div>
+            <div id="padding">
+            <a href="gallery.html"> Gallery </a>
+            </div>
+        </div>
+        </header>
+        <div id="container">
+        <p><img src="http://g.recordit.co/HbRJhGXNiX.gif" alt="app example" width="200px" height="400px">
```

Comment:
Not really something you ahve to change for this project, and I think it's fine to leave it as is, but FYI:

 alt texts should describe the content of the image. Screen readers will read the alt text aloud.

Reply:
solved